src/cart.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sabka Bazaar</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Dosis:400,600,700">
+</head>
+
+<body>
+    <header id="header"></header>
+    <section>
+        <div class="container">
+            <div class="onhover_Cart">
+                <div class="cart_Box">
+                    <div>
+                        <h3>My Cart<span>(1 item)</span></h3>
+                        <div class="cart_Items">
+                            <ul>
+
+                                <li>
+                                    <div class="items
+                                flex_box">
+                                        <div class="img_wrap">
+                                            <img src="static/images/products/fruit-n-veg/apple.jpg" alt="">
+                                        </div>
+                                        <div class="detail_wrap">
+                                            <h5>Fruits & Vegetable</h5>
+                                            <div>
```

Comment:
H5 is skip level heading. where is H1, H2 and H4 tags?

Reply:
Done